codegen/smithy-aws-ruby-codegen/src/main/java/software/amazon/smithy/aws/ruby/codegen/HttpBearerAuth.java

```diff
@@ -49,12 +49,18 @@ public List<ClientConfig> getAdditionalClientConfig(GenerationContext context) {
                 @see AWS::SDK::Core::CREDENTIALS_PROVIDER_CHAIN
                 """;
 
+        String identityProviderChain = "AWS::SDK::Core::HTTPBearerProviderChain";
```

Comment:
This feels fairly similar between identity types - we could create something generic for it.  Probably overkill for now since we really only have the 2 cases.

Reply:
What do you mean? I think this also differs from the python quip.

Reply:
Deferring work on the provider chain for bearer.